latest/clone-and-build.test.yml

```diff
@@ -0,0 +1,9 @@
+
+sut:
+  image: ${IMAGE_NAME}
+  command: sudo su duke
+           cd ~/dev && \
+           git clone https://github.com/DukeRobotics/robosub-ros.git && \
+           source /opt/ros/kinetic/setup.bash && \
+           cd robosub-ros/catkin_ws && \
+           catkin build
```

Comment:
The online image does not contain the catkin directory, so before `catkin build` should be `catkin init`

Reply:
By catkin directory I mean the .catkin_tools folder that signifies it is a catkin workspace.

Reply:
per the [docs for catkin build](https://catkin-tools.readthedocs.io/en/latest/verbs/catkin_build.html):

> If a workspace is not yet initialized, build can initialize it with the default configuration, but only if it is called from the workspace root.

I understand if you think we should initialise anyway for clarity though

Reply:
oh ok I did not know that, maybe we can still initialise for clarity

Reply:
Up to you